Classes/Protecta.uc

```diff
@@ -14,6 +14,35 @@ exec function SwitchModes()
 	DoToggle();
 }
 
+//overridden to fix zooming in and out
+simulated exec function ToggleIronSights()
+{
+	if( bHasAimingMode )
+	{
+		if( bAimingRifle )
+		{
+			PerformZoom(false);
+			TweenAnim(IdleAnim,ZoomTime/2); //fix zoom out
+		}
+		else
+		{
+            if( Owner != none && Owner.Physics == PHYS_Falling &&
```

Comment:
Indentation is messed up due to usage tabs + spaces. Please convert all tabs to 4 spaces in the entire file (including already existing code).

Reply:
Actually, nevermind. I found that there are indentation issues everywhere in ScrnWeaponPack. So I'll merge your changes and then fix indents myself.